test/integration/test.events-postroll.js

```diff
@@ -3,7 +3,6 @@ TODO:
 * timeupdate, adtimeupdate, contenttimeupdate
 * loadstart, adloadstart, contentloadstart
 * play, adplay, contentplay
-* contentended
```

Comment:
should we include `readyforpostroll` here too?

Reply:
These tests are primarily for redispatching, so really, contentended should stay on this list for the future when it's redispatched normally, rather than adding readyforpostroll here. Adding contentended back